services/FileService.js

```diff
@@ -0,0 +1,77 @@
+const multer = require("multer");
+const axios = require("axios").default;
+const FormData = require("form-data");
+const { unlinkSync, createReadStream } = require("fs");
+const { extname } = require("path");
+
+const Image = require("../models/Image");
+
+/** Maximum image count */
+const MAX_IMAGE_COUNT = 3;
+/** Destination for storing images */
+const IMAGE_DESTINATION = "./images";
+
+/** Multer Storage Engine, configured to store files locally */
+const storage = multer.diskStorage({
+  destination: IMAGE_DESTINATION,
+  filename: (_, { originalname }, cb) => {
+    cb(null, `${originalname}${extname(originalname)}`);
```

Comment:
Do you think we should be naming it this way? Cuz if there are multiple images with the same name, they might get overridden, which can be a possibility during multiple requests.

Reply:
It was just to time being. As u earlier said, it would be hashed

Reply:
Make that change then before we merge in.